core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-properties-form/field-properties/dot-relationships-property/dot-relationships-property.component.ts

```diff
@@ -41,6 +41,23 @@ export class DotRelationshipsPropertyComponent implements OnInit {
         this.editing = !!this.group.get(this.property.name).value.velocityVar;
     }
 
+    /**
+     * check field is required or not
+     * @readonly
+     * @type {boolean}
+     * @memberof DotRelationshipsPropertyComponent
+     */
+    get isRequired(): boolean {
+        const validatorFunc = this.group.get(this.property.name).validator;
+        if (validatorFunc) {
+            const validator = validatorFunc({} as AbstractControl);
+
+            return validator && validator.required;
+        }
+
+        return false;
+    }
+
```

Comment:
I was also thinking for the `.pipe` but after discussion with @fmontes  we decided to keep it simple, but I think `.pipe` is best for this case because we only need to pass `formControl` in pipe and can check easily if it is required or not. What do you think @fmontes?